### PR TITLE
[PLAY-1145] Move Icon Page

### PR DESCRIPTION
**Pull request by markdoeswork**
**What does this PR do?** A clear and concise description with your runway ticket url.

Runway: https://nitro.powerhrg.com/runway/backlog_items/PLAY-1145

**Screenshots:** Screenshots to visualize your addition/change
screenshot-127 0 0 1_3000-2024 01 23-15_28_47

**How to test?** Steps to confirm the desired behavior:
1. Go to https://pr3072.playbook.beta.gm.powerapp.cloud/beta/icons
2. witness icons

#### Checklist:
- [😈] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [😈] **TESTS** I have added test coverage to my code.

### CODE REVIEW

**Review by thestephenmarshall — APPROVED** [review]
Code looks good to me. I tested `/beta/icons` in Milano and it is rendering correctly.

File: playbook-website/app/javascript/components/Website/src/views/Icons.tsx
Comment (jasperfurniss) [inline playbook-website/app/javascript/components/Website/src/views/Icons.tsx:1]: Let's remove this file @markdoeswork

File: playbook-website/app/javascript/components/Website/src/views/README.md
Comment (jasperfurniss) [inline playbook-website/app/javascript/components/Website/src/views/README.md:1]: Please delete this added file.

**Review by jasperfurniss — CHANGES_REQUESTED** [review]
@markdoeswork Go ahead and delete those extra files you had in here... looks like we added a views directory. But I think you realized we had one called "pages" already. 

Another piece of feedback from product is this: We can go ahead and remove the route that is on the legacy (current) site for `/icons`.... 
So we will _only_ work in the `beta/icons` for the time-being, until we are ready for launch. Thanks @markdoeswork

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**